master/reference/calicoctl/resources/policy.md

```diff
@@ -65,6 +65,52 @@ the policy is enforced after connection tracking and any DNAT.
+latter case meaning to deny ingress by default - and to egress traffic (from
+the selected endpoints) only if it has any egress rules.
+
+| Ingress rules present? | Egress rules present? | Default `types`   | Other allowed `types` settings, and their meaning                                                     |
+|------------------------+-----------------------+-------------------+-------------------------------------------------------------------------------------------------------|
+| No                     | No                    | `ingress`         | `egress` - Restrict egress instead of ingress.  `ingress, egress` - Restrict both ingress and egress. |
+|------------------------+-----------------------+-------------------+-------------------------------------------------------------------------------------------------------|
+| Yes                    | No                    | `ingress`         | `ingress, egress` - Restrict egress as well as ingress.                                               |
+|------------------------+-----------------------+-------------------+-------------------------------------------------------------------------------------------------------|
+| No                     | Yes                   | `egress`          | `ingress, egress` - Restrict ingress as well as egress.                                               |
+|------------------------+-----------------------+-------------------+-------------------------------------------------------------------------------------------------------|
+| Yes                    | Yes                   | `ingress, egress` | None.                                                                                                 |
+|------------------------+-----------------------+-------------------+-------------------------------------------------------------------------------------------------------|
+
+> **NOTE**
+>
+> It is not valid to specify a `types` value _without_ `ingress` when ingress
+> rules are present, or to specify a `types` value _without_ `egress` when
+> egress rules are present.
+>
+> Prior to v2.6.0, Calico behaved as though `types` was always `[ ingress,
+> egress ]`.  In other words, if a policy applied to a given endpoint at all,
+> it did so for both ingress and egress traffic, even if that policy specified,
+> for example, no egress rules.  However this made it impossible to define
+> ingress policy for some endpoints without simultaneously affecting the
+> default egress policy for them, and vice versa; so in Calico v2.6.0 the
+> `types` field was introduced and the default behavior changed.
+>
+> If you have existing YAML with policies without any ingress or egress rules,
+> that you want to behave as they did prior to Calico v2.6.0, simply add
+> `types: [ ingress, egress ]` to each such policy.
+
 #### Rule
 
 | Field       | Description                 | Accepted Values   | Schema | Default    |
```

Comment:
The "Default" column is actually empty. It does not contain anything. You should either delete this column or provide the default value....I would recommend the latter, if possible!